test/pkcs11_mbedtls_utest/core_pkcs11_mbedtls_utest.c

```diff
@@ -356,6 +366,8 @@ static void prvCommonInitStubs( CK_SESSION_HANDLE_PTR pxSession )
     TEST_ASSERT_EQUAL( CKR_OK, xResult );
     xResult = prvOpenSession( pxSession );
     TEST_ASSERT_EQUAL( CKR_OK, xResult );
+
+    mbedtls_pk_get_type_Stub( pk_get_type_stub );
```

Comment:
I didn't get the point of doing this.
We are not checking the value nor are we using it. Did I miss something?

Reply:
The number of times which mbedtls_pk_get_type gets called is kinda irrelevant so I stubbed it globally rather than in each individual test case. 

It doesn't make much sense to mock out mbedtls when testing core_pkcs11_mbedtls.c since we more or less have to stub all the parts of the api which are used.

This whole unit test should probably just be replaced with an improved integration test, but that is a much larger effort.

Reply:
Thanks for the explanation.

But, I must confess that, I was not thinking properly when I wrote that comment. It is pretty clear to me now what this is used for.